### ISSUE TITLE
🔍 Accessibility Issue: page-has-heading-one - MODERATE Impact

### ISSUE THREAD

**Issue by FarukDelic** [FarukDelic/Axe-Python-Demo#5]
## Accessibility Violation Found
    
**Rule ID:** `page-has-heading-one`  
**Impact:** MODERATE  
**URL Tested:** https://www.google.com  
**Description:** Ensure that the page, or at least one of its frames contains a level-one heading

### Help Information
Page should contain a level-one heading

**Help URL:** https://dequeuniversity.com/rules/axe/4.4/page-has-heading-one?application=axeAPI

#### Element 1
- **HTML:** `<html itemscope="" itemtype="http://schema.org/WebPage" lang="bs">`
- **Target:** `html`
- **Failure Summary:** Fix all of the following:
  Page must have a level-one heading
- **All of these checks failed:**
  - Page must have a level-one heading

### How to Fix
Please refer to the [Axe Rules documentation](https://dequeuniversity.com/rules/axe/4.4/page-has-heading-one?application=axeAPI) for detailed guidance on resolving this accessibility issue.

---
*This issue was automatically created by the Axe Accessibility Test workflow.*
